File: push/src/main/java/top/trumeet/mipushframework/wizard/WelcomePhonyPermissionInfo.java
package top.trumeet.mipushframework.wizard;

import android.content.Context;

import androidx.annotation.NonNull;

import com.xiaomi.xmsf.R;

public class WelcomePhonyPermissionInfo extends DisplayOnlyPhonyPermissionInfo {
    public WelcomePhonyPermissionInfo(Context context) {
        super(context);
    }

    @NonNull
    @Override
    public String getPermissionTitle() {
        return context.getString(R.string.app_name);
    }

    @NonNull
    @Override
    public String getPermissionDescription() {
        return context.getString(R.string.wizard_descr);
    }

}

File: xmsf/src/main/java/io/github/magisk317/mipush/feature/wizard/support/WelcomePhonyPermissionInfo.kt
package io.github.magisk317.mipush.feature.wizard.support

import android.content.Context
import com.xiaomi.xmsf.R

class WelcomePhonyPermissionInfo(context: Context) : DisplayOnlyPhonyPermissionInfo(context) {
    override val permissionTitle: String
        get() = context.getString(R.string.app_name)

    override val permissionDescription: String
        get() = context.getString(R.string.wizard_descr)
}
